packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -196,48 +196,9 @@ If a value with a unit would be formatted with exponential notation, it will alw
 
-Unit elements will be capable of enumerating the individual units supported. The enumerated unit objects will be capable of formatting a given number into a localized string including the unit label. Below is an example of the abstractions/APIs that might be used to implement this.
+We will use `nimble-unit-byte` to display file sizes in SLE tables. Currently, SLE displays these values with the common `KB`/`MB`/`GB` unit labels, but uses a factor of 1024 to convert between units (which is [not uncommon](https://en.wikipedia.org/wiki/JEDEC_memory_standards#Unit_prefixes_for_semiconductor_storage_capacity), but [technically incorrect](https://physics.nist.gov/cuu/Units/binary.html)). In [ADR33: Byte Representation in Stratus](https://dev.azure.com/ni/DevCentral/_git/Skyline?path=/docs/architecture-decisions/adr0033-byte-representation-in-Stratus.md&_a=preview&version=GBmaster) it was decided that SLE will migrate to base 10 byte prefixes.
 
-```TS
-// a specific unit, e.g. kilobyte, millivolt, etc.
-interface ScaledUnit {
-    public conversionFactor: number;
-    public format(value: number): string;
-}
-
-// a set of related units, e.g. {byte, kilobyte, megabyte, gigabyte, terabyte, petabyte}
-interface UnitScale {
-    public getSupportedUnits(lang: string, formatterOptions: Intl.NumberFormatOptions): ScaledUnit[];
-}
-
-// unit scale supported by Intl.NumberFormat
-class UnitByte extends FoundationElement implements UnitScale {
-    public getSupportedUnits(lang: string, formatterOptions: Intl.NumberFormatOptions): ScaledUnit[] {
-        // returns implementations of ScaledUnit that wrap Intl.NumberFormat instances configured for a specific locale and a specific unit (e.g. 'fr-FR' and 'kilobyte')
-    }
-}
-
-// unit scale with Nimble-provided unit translations
-class UnitVolt extends FoundationElement implements UnitScale {
-    public getSupportedUnits(lang: string, formatterOptions: Intl.NumberFormatOptions): ScaledUnit[] {
-        // returns implementations of ScaledUnit that contain a shared Intl.NumberFormat (for formatting the number) and a specific translated unit string to append
-    }
-}
-```
-
-We will use `nimble-unit-byte` to display file sizes in SLE tables. Currently, SLE displays these values with the common `KB`/`MB`/`GB` unit labels, but uses a factor of 1024 to convert between units (which is [not uncommon](https://en.wikipedia.org/wiki/JEDEC_memory_standards#Unit_prefixes_for_semiconductor_storage_capacity), but [technically incorrect](https://physics.nist.gov/cuu/Units/binary.html)). Whether SLE chooses to standardize on the default 1000-based byte units or the 1024-based ones, it will be a change from the current behavior. To ensure consistency, we will update SLE's file size pipe and search for other places where byte values are being converted so they can be updated accordingly.
-
-##### Creating new units
```

Comment:
Do we not want this section? I'm indifferent--I think it's simple enough to follow the examples of existing units--but I think @jattasNI had asked for this to be documented.

Reply:
Didn't find the unit element discussion very necessary, They are pretty trivial.

I tweaked the unit-format README to be a bit clearer about the different unit-scale types. I think it captures everything that was here but I've also looked at it too long so up for adding additional docs if @jattasNI can think of some that are useful.

Reply:
@jattasNI did you want to make a comment on this specifically?

Reply:
If you're okay with the deletion, resolve this thread, so the auto-merge can complete.

Reply:
@jattasNI I'll go ahead and resolve for now to move the PR, can add additional docs on a follow up if you have additional feedback